aocd/runner.py

```diff
@@ -20,17 +22,24 @@
 from .utils import AOC_TZ
 from .utils import colored
 from .utils import get_plugins
+from ._types import _Part
 
+if TYPE_CHECKING:
+    if sys.version_info >= (3, 10):
+        from typing import ParamSpec
+    else:
+        from typing_extensions import ParamSpec
+    _PS = ParamSpec("_PS")
```

Comment:
move the logic into a `compat.py` submodule so there is an unconditional import of these names.